tgui/packages/tgui/styles/themes/ghost.scss

```diff
@@ -0,0 +1,5 @@
+.theme-ghost {
+  .Layout__content {
+    background-image: none;
+  }
+}
```

Comment:
Ok I'll bite.

Do we really need a new theme just to hide the NT logo? 

Even if we do keep it, why not just make it `nologo` to reduce confusion.

Reply:
It looked pretty terrible with the logo. And unfortunately, this is the only way to override it with how the system is currently set up. As for the name, no problem changing that.

Reply:
Yeah. Make it `nologo`. Although the PDA one looks fine with the logo so hmmmmmmmmmmm.